instrumentation/delayed_job/test/test_helper.rb

```diff
@@ -24,31 +29,8 @@
   c.add_span_processor span_processor
 end
 
-ActiveRecord::Migration.verbose = false
-
-module TestHelper
-  extend self
-
-  def setup_active_record
-    ::ActiveRecord::Base.establish_connection(adapter: 'sqlite3', database: ':memory:')
-    ::ActiveRecord::Schema.define do
-      create_table 'delayed_jobs', force: :cascade do |t|
-        t.integer 'priority', default: 0, null: false
-        t.integer 'attempts', default: 0, null: false
-        t.text 'handler', null: false
-        t.text 'last_error'
-        t.datetime 'run_at'
-        t.datetime 'locked_at'
-        t.datetime 'failed_at'
-        t.string 'locked_by'
-        t.string 'queue'
-        t.datetime 'created_at'
-        t.datetime 'updated_at'
-      end
-    end
-  end
-
-  def teardown_active_record
-    ::ActiveRecord::Base.connection.close
-  end
-end
+gem_dir = Gem::Specification.find_by_name('delayed_job').gem_dir
+require "#{gem_dir}/spec/delayed/serialization/test"
```

Comment:
Any reason to load this empty file?

https://github.com/collectiveidea/delayed_job/blob/v4.1.11/spec/delayed/serialization/test.rb

Reply:
This may not be required since I am using the class instead of a symbol. I will remove it and see if tests pass. 

https://github.com/collectiveidea/delayed_job/blob/master/lib/delayed/worker.rb#L67

Reply:
I removed this line and the tests pass.